service/orchestrator/assessment_results_test.go

```diff
@@ -83,20 +82,23 @@ func TestService_GetAssessmentResult(t *testing.T) {
 			},
 		},
 		{
-			name: "record found in database",
+			name: "permission denied because of non authorized cloud_service_id",
 			fields: fields{
 				storage: testutil.NewInMemoryStorage(t, func(s persistence.Storage) {
 					assert.NoError(t, s.Create(&orchestratortest.MockAssessmentResult1))
 					assert.NoError(t, s.Create(&orchestratortest.MockAssessmentResult2))
+					assert.NoError(t, s.Create(&orchestratortest.MockAssessmentResult3))
 				}),
 				authz: servicetest.NewAuthorizationStrategy(false, []string{orchestratortest.MockAssessmentResult1.CloudServiceId}),
 			},
 			args: args{
-				req: orchestratortest.MockAssessmentResultRequest1,
+				req: orchestratortest.MockAssessmentResultRequest2,
+				ctx: testutil.TestContextOnlyService1,
+			},
+			res: nil,
+			wantErr: func(t assert.TestingT, err error, i ...interface{}) bool {
+				return assert.Contains(t, err.Error(), "access denied")
```

Comment:
```suggestion
				return assert.ErrorContains(t, err, service.ErrPermissionDenied.Error())
```

Reply:
Would be more safe if, e.g., the message of `ErrPermissionDenied` would be changed in the future